### PR TITLE
[Keymap] Add kifinnsson keymap for dz60

### PR DESCRIPTION
**Pull request by kifinnsson**
## kifinnsson's keymap for the DZ60 Keyboard

Keymap for my non-standard DZ60 layout. It is an ansi layout on the right and iso on the left (ie 1.25x left shift). This is to implement the angle mod on for Colemak which is the base layer. A side effect of this is that I have an extra key on row 4, which sits between the "b" and "k" keys in Colemak. I use this key as a switch to layer 2 which is my macro layer. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
It looks like you have a number of issues here.  You may want to see about rebasing your branch, as that may clean up a bunch of the issues. 

https://docs.qmk.fm/#/newbs_best_practices

This should get you started/pointed in the right direction.

File: keyboards/dz60/keymaps/kifinnsson/rules.mk [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# MCU name
+MCU = atmega32u4
```
Comment (mechmerlin) [inline keyboards/dz60/keymaps/kifinnsson/rules.mk]: Like the `config.h` file this `rules.mk` in the keymaps directory only needs to contain things that are different from the keyboard's `rules.mk`.

File: keyboards/dztech/dz60rgb/rules.mk [hunk 2]
```diff
@@ -1,6 +1,48 @@
+<<<<<<< HEAD
```
Comment (mechmerlin) [inline keyboards/dztech/dz60rgb/rules.mk]: Looks to be some merge conflict artifacts here

File: keyboards/gergo/keymaps/drashna/keymap.c [hunk 3]
```diff
@@ -9,17 +9,34 @@
 #include QMK_KEYBOARD_H
 #include "drashna.h"
 
+<<<<<<< HEAD
```
Comment (mechmerlin) [inline keyboards/gergo/keymaps/drashna/keymap.c]: Looks like some merge conflict artifacts here

**Comment by kifinnsson** [comment]
Yep, I'll get it all cleaned up! Thanks!

**Pull request closed without merging**